act-rules-format.bs

```diff
@@ -52,66 +48,91 @@ The separation between atomic rules and composite rules creates a division of re
     <li>Header indicated by using the `headers` attribute</li>
     <li>Header indicated by using ARIA labels</li>
   </ul>
-  <p>If any one of these rules passes, the test result of the composite rule for the table cell is "passed".</p>
+  <p>If any one of these rules passes, the outcome of the composite rule for the table cell is "passed".</p>
 </div>
 
+The separation between atomic rules and composite rules creates a division of responsibility. Atomic rules test if web content is correctly implemented in a particular solution. Composite rules test if a combination of outcomes from atomic rules is sufficient for an accessibility requirement to be conformant or not.
```

Comment:
"Atomic rules test if web content is correctly implemented in a particular solution" -> "Atomic rules test if test subjects correctly implement particular solutions" or "Atomic rules test for particular solutions within test subjects"

"Composite rules test if a combination of outcomes from atomic rules is sufficient for an accessibility requirement to be conformant or not." -> "Composite rules test if a combination of outcomes from atomic rules satisfy/meet the conformance for an accessibility requirement."